conda_recipes/aftereffects-25.0/recipe/meta.yaml

```diff
@@ -0,0 +1,27 @@
+{% set name = "aftereffects" %}
+{% set version_partial = "25.0" %}
+{% set version_minor = "1" %}
```

Comment:
Is the 0 in 25.0 not the minor version and 1 is the patch version?

Reply:
hmm you are right. Will update.